submissions/goodnessOlakanmi/html-forms/Contact-Form-Page.html

```diff
@@ -6,8 +6,8 @@
 </head>
 <body>
 	<h1>Contact Form</h1>
-		<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Contact-Form-Page.html">Contact Form</a></li>
-		<li><a href="C:\Users\USER\Desktop\Dufuna-Fem\submissions\goodnessOlakanmi\html-forms\Product-page.html">Product Page</a></li>
+		<li><a href="Contact-Form-Page.html">Contact Form</a></li>
+		<li><a href="Product-page.html">Product Page</a></li>
```

Comment:
Well done. But this link won't work for the product page. Something is still missing

Reply:
Oh really! But it worked when I checked it. Please what do you think is wrong with it?

Reply:
I have worked on it. Thank you.